assets/together.yaml

```diff
@@ -146,7 +146,7 @@
   training_hardware: ''
   quality_control: ''
   access: open
-  license: Apache 2.0
+  license: LLaMA 2
```

Comment:
change to Llama 2 

LLaMA is typeset in this capitalized way, but Llama 2 is typeset in title case